index.html

```diff
@@ -140,16 +140,17 @@ <h2>About me</h2>
       <p>
         I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
       </p>
-      <form action="https://formspree.io/f/xbjezgwz" method="POST">
+      <form action="https://formspree.io/f/xbjezgwz" method="POST" onsubmit="return emailValidation()">
```

Comment:
On successful submission of the form, the data should still be sent to the formspree service. Currently, only the 'messege sent' text is being shown. Please take a look at this🙏